### PR TITLE
Rename wormchain

### PR DESCRIPTION
**Pull request by conorpp**
We have inconsistent naming for wormchain: `wormholechain`, `wormhole-chain`, and `wormchain`.  `wormchain` is definitely the name I've heard everyone use and just wanted to make it consistent while we still can.

### CODE REVIEW

**Review by SEJeff — COMMENTED** [review]
hero

File: clients/js/main.ts [hunk 1]
```diff
@@ -883,8 +883,9 @@ yargs(hideBin(process.argv))
           argv["contract-address"],
           argv["rpc"]
         );
-      } else if (chain === "wormholechain") {
-        throw Error("Wormhole Chain is not supported yet");
+      } else if (chain == "wormholechain" || (chain+"") === "wormchain") {
```
Comment (jynnantonix) [inline clients/js/main.ts:886]: javascript lol

**Review by jynnantonix — DISMISSED** [review]

**Review by hendrikhofstadt — APPROVED** [review]
@evan-gray tilt fails on this because it does not see the SDK updates. I assume we can merge this and the cut a release / import that one?

**Comment by conorpp** [comment]
I think we should merge and update to a new release of the js sdk.  I believe most places have backwards compat, the only spot that is breaking currently is the clients/js/main.ts tool.

**Review by kev1n-peters — APPROVED** [review]

**Comment by evan-gray** [comment]
Overriding on the assurances of cutting an sdk release and fixing up the client with it. One day we'll switch to [Lerna](https://lerna.js.org/)

**Pull request merged**